jest.config.js

```diff
@@ -0,0 +1,188 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
We don't really need all of this commented out stuff…

Reply:
I will delete them.